### ISSUE TITLE
Argument parser errors are printed without a trailing newline

### ISSUE THREAD

**Issue by emanuele6** [hhatto/autopep8#713]
Command Line
```console
$ autopep8
incorrect number of arguments$ autopep8
incorrect number of arguments$
```

## Your Environment
* Python version: `Python 3.11.5`
* autopep8 version: `autopep8 2.0.4 (pycodestyle: 2.11.0)`
* Platform: windows, linux, macOSX, and other OS...: archlinux

Errors are printed without a trailing newline

**Comment by emanuele6** [comment]
I bisected the problem; it was introduced with 24da5065e1fefcf276f60412e90491b578778fde that replaced all calls to `parser.error(message)` to calls to `parser.exit(EXIT_CODE_ARGPARSE_ERROR, message`.

Before the change, those argparse errors were displayed as:
```
$ ./autopep8.py 
usage: autopep8 [-h] [--version] [-v] [-d] [-i] [--global-config filename]
                [--ignore-local-config] [-r] [-j n] [-p n] [-a]
                [--experimental] [--exclude globs] [--list-fixes]
                [--ignore errors] [--select errors] [--max-line-length n]
                [--line-range line line] [--hang-closing] [--exit-code]
                [files ...]
autopep8: error: incorrect number of arguments
```
Now, they are disaplyed as:
```
$ ./autopep8.py
incorrect number of arguments$ # no trailing newline
```

**Comment by hhatto** [comment]
Thanks for reporting 👍 

fix and v2.3.1 has been released.
https://pypi.org/project/autopep8/2.3.1/
